Assets/Scripts/TranslationService/API/TranslationAPI.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace TranslationService
+{
+    /// <summary>
+    /// API of the Translation Service. We should contact this API via the Service associated.
+    /// The API Contains method that can be used by other services.
+    /// </summary>
+    public class TranslationAPI
+    {
+        private TranslationEngine engine;
+
+        public TranslationAPI(TranslationEngine engine)
+        {
+            this.engine = engine;
+        }
+
+        /// <summary>
+        /// Retrieve the localized string from the unlocalized one (the key).
+        /// </summary>
+        /// <param name="key"></param>
+        /// <returns>The Localized String associated to the key, from the current language.</returns>
+        public string GetLocalizedString(string key)
+        {
+            return engine.UnocalizedStrings.ContainsKey(key) ? engine.UnocalizedStrings[key] : TranslationConstants.ERROR_KEY_PATTERN + key;
```

Comment:
If the key exists, we return the value of the key. If it does not, we return the key itself, preceded by a tag #ERROR_$.
For example, imagine that in the json we have something like : 
`	{
		"key":"Hello",
		"value":"Bonjour"
	}`

Then, if we call `GetLocalizedString("Hello")`, that will return `"Bonjour"`.
If we call `GetLocalizedString("AnUnknownKey")` then, that will return `"#ERROR_$AnUnknownKey"`

Be careful because the key is case sensitive ("Hello" and "hello" are two differents keys).

Reply:
Thank's a lot, it was that i supposed to.